### DISCUSSION TITLE
Primera semana de Pensamiento Computacional y he entendido...

### DISCUSSION THREAD

**Discussion by escuelaDeCodigoMargaritaMaza** [escuelaDeCodigoMargaritaMaza/escuela_de_codigo#5] [General]
Qué te llevas esta semana?

**Comment by OmarJoasAD** [comment]
Mucho aprendizaje de lo que hemos visto ésta semana... Algoritmos, Operadores, Tipos de Datos y los Conceptos Básicos de Programación, todo antes de empezar a programar. Todo muy importante e interesante a la vez. Me gusta. 👍🏻👍🏻👏🏻 Todo ésto me llevo de la semana.

**Comment by RenRen991027** [comment]
Sin lugar a dudas continuaré yendo a las clases, el compañero nos ha dejado muy claros los conceptos con sus explicaciones. Esta semana aprendí rápidamente los conceptos de _algoritmo, tipos de datos, los operadores y los conceptos más básicos de programación_, todo esto me ayuda y me motiva a continuar mi camino adentrándome en el campo de la programación.

**Comment by Darki151101** [comment]
Mucho conocimiento nesesario para empezar a programar, tanmbien un buen ambiente y las clases que dan son cencilla pero practicas faciles de enteder voy a seguri llendo

**Comment by mavijg** [comment]
El comprender y entender mejor los conceptos de tipo de dato y el usar de forma más simple los operadores.

**Comment by Iscrack** [comment]
En esta semana he comprendido los conceptos de algoritmos, operadores y lenguajes de programación, así como las clasificaciones, funciones y usos de cada uno. Cada tema se complementa con ejercicios en clase, lo que aporta claridad y solidez al contenido abordado.

**Comment by robertodegaona** [comment]
esta semana he aprendido lo que es un algoritmo, los tipos de operadores que vamos a ocupar, los tipos de datos y lo básico para programar

**Comment by Danniela32** [comment]
Lo que aprendí durante la primer semana fue que a partir del pensamiento computacional podemos dividir un problema, reconocer patrones, prever situaciones y crear instrucciones a base de algoritmos. También pude aprender los tipos de operadores que existen, entendí lo que es un lenguaje de programación y que existen tipos de datos los cuales se expresan con palabras diferentes.

**Comment by Citlalli21** [comment]
Esta semana me llevo muchos conceptos de lo básico en programación, también cómo resolver problemas a través de un algoritmo, el uso correcto de los operadores y los diferentes tipos de datos que hay. El que haya práctica de cada tema, y que el compañero David resuelva todas nuestras dudas, hace que me vaya familiarizando con éste nuevo lenguaje.

**Comment by Jahlonsoquai** [comment]
Antes que nada he aprendido la política de como se maneja el trato entre la comunidad que conforma a los PILARES, una filosofía de cooperativismo y aportación por y para los compañeros. En cuanto al programa, veo que el compañero David domina el tema y se nota que es un buen instructor. Lo que me llevo de aprendizaje sustancialmente, es la importancia de los algoritmos para realizar las indicaciones o series de pasos que son indispensables en la programación; los tipos de operadores que hay tanto aritméticos, relacionales y lógicos, que hacen funcionar la maquinaria de el código en los distintos tipos de lenguaje de programación (tanto los de alto y bajo nivel) y que son ejecutables para que la computadora  procese en lenguaje máquina el programa que se desea tener.

**Comment by Alancin01** [comment]
Lo que aprendí de la escuela de código es comprender de una manera sencilla, los logaritmos y significado de los programas, sin tener que ser pesado el tratar de entenderlo, el ritmo del aprendizaje es bueno y poco a poco me estoy interesando cada vez más en esto de la programación, y espero las siguientes semanas sean iguales

**Comment by Alastorstark** [comment]
la solucion de  problemas mediante pasos  establecidos para la solucion mas rapida y mas simple como funcionan los algoritmos  y se me  hace interesante saber el por que de cada algoritmo
